Sources/Kitura/RouterRequest.swift

```diff
@@ -184,6 +185,21 @@ public class RouterRequest {
         _ = try serverRequest.read(into: &data)
         return try JSONDecoder().decode(type, from: data)
     }
+    
+    /// Read the URLEncoded body of the request as a Codable object.
+    ///
+    /// - Parameter type: Codable object to which the body of the request will be converted.
```

Comment:
This parameter is called `as` not `type` (the doc is wrong on the function above too...)